Classes/CAPSPageMenu.swift

```diff
@@ -135,53 +136,63 @@ public class CAPSPageMenu: UIViewController, UIScrollViewDelegate, UIGestureReco
         
         if options != nil {
             for key : String in options!.keys {
-                if key == "selectionIndicatorHeight" {
+                switch key {
+                case "selectionIndicatorHeight":
                     selectionIndicatorHeight = options![key] as CGFloat
-                } else if key == "menuItemSeparatorWidth" {
+                case "menuItemSeparatorWidth":
```

Comment:
You can also use enums like :
`fileprivate enum somthing: String {
       case menuItemSeparatorWidth = "menuItemSeparatorWidth"
       case scrollMenuBackgroundColor = "scrollMenuBackgroundColor"
}`